eng/pipelines/libraries/build-job.yml

```diff
@@ -164,6 +168,14 @@ jobs:
               artifactName: $(librariesBuildArtifactName)
               displayName: Build Assets
 
+          # Save AllConfigurations artifacts using the prepare-signed-artifacts format. The
```

Comment:
We could avoid publishing all bin directory by conditioning all the steps above when targetFramework eq allConfigurations. Basically adding that condition to the current condition where runTests eq false.

With that we no longer need `_skipTestHostCopy` variable.

Reply:
I mean, the only needed assets in an official build from an allCOnfigurations leg are the packages, which you're uploading as part of this step.

As follow up of the clean-up, we should be able to remove these, and just upload the packages as part of the artifacts asset.

Reply:
Good point, I've added a note that this particular publish step goes away when we un-isolate the allconfigurations artifacts.

Reply:
Thanks for adding the note. Would you mind conditioning all the copy steps above and the upload of those artifacts with a compile time condition expanding the current one to check for framework eq allConfigurations?

Reply:
I'd rather leave that until later too... I'm not really familiar with when those copies are used and don't quite follow the change. It seems like incremental improvement to me. This PR is also very close to green after a few hangs, would be nice to get it merged without tinkering with this bit. I'll definitely add it to the note though.

Reply:
Ok sounds good. We have it as part of the issue so we can cleanup once we get our hands to that issue.